testing/code/test_excinfo.py

```diff
@@ -755,14 +764,18 @@ def f():
         assert tw.lines[0] == "    def f():"
         assert tw.lines[1] == ">       g(3)"
         assert tw.lines[2] == ""
-        assert tw.lines[3].endswith("mod.py:5: ")
```

Comment:
@suzaku just a question, why did you changed those tests? Your change to the terminal writer required those to be changed?